samples/extensions/open_cl_interop/open_cl_interop.cpp

```diff
@@ -0,0 +1,622 @@
+/* Copyright (c) 2021, Arm Limited and Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "open_cl_interop.h"
+
+#include "common/vk_common.h"
+#include "gui.h"
+#include "platform/filesystem.h"
+#include "platform/platform.h"
+
+#define CL_FUNCTION_DEFINITIONS
+#include <open_cl_utils.h>
+
+struct CLData
+{
+	cl_context       context{nullptr};
+	cl_device_id     device_id{nullptr};
+	cl_command_queue command_queue{nullptr};
+	cl_program       program{nullptr};
+	cl_kernel        kernel{nullptr};
+	cl_mem           image{nullptr};
+};
+
+OpenCLInterop::OpenCLInterop()

```

Comment:
index_count should be initialized here.  I also recommend using initializer lists for zoom, title etc.